src/coreclr/jit/helperexpansion.cpp

```diff
@@ -744,7 +744,7 @@ bool Compiler::fgExpandThreadLocalAccessForCallNativeAOT(BasicBlock** pBlock, St
     fastPathBb->inheritWeight(prevBb);
 
     // fallback will just execute first time
-    fallbackBb->bbSetRunRarely();
+    fallbackBb->inheritWeightPercentage(tlsRootNullCondBB, 0);
```

Comment:
It looks a bit silly, but unlike `bbSetRunRarely`, `inheritWeightPercentage` propagates the `BBF_PROF_WEIGHT` flag from the block its deriving weight from. Changing `bbSetRunRarely` to propagate the flag would likely incur large diffs in the non-PGO case.